pkg/attestation/crafter/api/attestation/v1/crafting_state.proto

```diff
@@ -146,6 +146,9 @@ message PolicyEvaluation {
   // Evaluation messages, intended to communicate evaluation errors (invalid input)
   repeated string skip_reasons = 14;
 
+  // Group this evaluated policy belongs to, if any
+  string group_name = 15;
```

Comment:
I thought we were going to add some sort of group-ref?

Reply:
Right. I'll set this on draft.

Reply:
Done, please review again